Makefile

```diff
@@ -28,7 +28,7 @@ endif
 
 ifdef EMSCRIPTEN
 	# Todo #--js-library webassembly/helper.js
-	LDFLAGS+=--shell-file webassembly/x16emu-template.html  --preload-file rom.bin --preload-file chargen.bin -s DISABLE_DEPRECATED_FIND_EVENT_TARGET_BEHAVIOR=1 -s USE_PTHREADS=1 
+	LDFLAGS+=--shell-file webassembly/x16emu-template.html  --preload-file rom.bin --preload-file chargen.bin -s DISABLE_DEPRECATED_FIND_EVENT_TARGET_BEHAVIOR=1 
```

Comment:
The `chargen.bin` file is now included in `rom.bin`: https://github.com/commanderx16/x16-emulator/pull/64#issuecomment-533674910

Reply:
Will address , thank you